src/app/modules/app/pages/meetings/meetings.component.ts

```diff
@@ -1,8 +1,54 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { MEETING_HEADERS } from './meeting.constants';
+import { Meeting } from 'src/app/shared/models/Meetings';
+import { MeetingService } from 'src/app/shared/services/meeting.service';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { API_ERROR_MESSAGE } from 'src/app/shared/constants/api.constants';
+import { SNACKBAR_ERROR_DEFAULTS } from 'src/app/shared/constants/snackbar.constants';
 
 @Component({
   selector: 'app-meetings',
   templateUrl: './meetings.component.html',
   styleUrls: ['./meetings.component.sass'],
 })
-export class MeetingsComponent {}
+export class MeetingsComponent implements OnInit {
+  headers: string[] = MEETING_HEADERS;
+  data: Meeting[] = [];
+  isLoading = true;
+
+  constructor(
+    public meetingService: MeetingService,
+    private snackBar: MatSnackBar
+  ) {}
+
+  ngOnInit() {
+    this.fetchData();
+  }
+
+  private fetchData(): void {
+    this.isLoading = true;
+    this.meetingService.getAll().subscribe({
+      next: this.handleSuccess.bind(this),
+      error: this.handleError.bind(this),
+    });
+  }
+
+  private handleSuccess(data: Meeting[]): void {
+    console.log(data);
+    this.data = data;
```

Comment:
Remove Console log
Maybe we should avoid using "data" word. Everything can be "data".
Here "data" could be meetings or something similar